nixos/modules/security/apparmor.nix

```diff
@@ -200,10 +200,7 @@ in
           sed '1,/\[qualifiers\]/d' $footer >> $out
         '';
 
-    boot.kernelParams = [
-      "apparmor=1"
-      "security=apparmor"
-    ];
+    security.lsm = [ "apparmor" ];
```

Comment:
Are these equivalent?

Reply:
Yes, `apparmor=1` and `security=apparmor` are the legacy ways to do it. `lsm` allows for selecting multiple LSM's at boot time.

Reply:
https://docs.kernel.org/6.14/admin-guide/kernel-parameters.html

Reply:
I've traced this to at least kernel 6.1

Reply:
Sounds good. As pinged about, we should still add apparmor=1 because even though lsm=apparmor overrides the security= parameter we should still ask apparmor to activate itself.

Reply:
Looking at the [Arch Wiki](https://wiki.archlinux.org/title/AppArmor) it seems to just recommend lsm=apparmor. However, I can't tell if they don't have to specify apparmor=1 because it's enabled in Kconfig. Seems like we should specify it to be safe.

Reply:
We should check `security/apparmor/Kconfig` since that's what the AppArmor option references.

Reply:
Done